Source/WebGPU/WebGPU/Adapter.mm

```diff
@@ -147,7 +147,7 @@ size_t wgpuAdapterEnumerateFeatures(WGPUAdapter adapter, WGPUFeatureName* featur
     return WebGPU::fromAPI(adapter).enumerateFeatures(features);
 }
 
-bool wgpuAdapterGetLimits(WGPUAdapter adapter, WGPUSupportedLimits* limits)
+WGPUBool wgpuAdapterGetLimits(WGPUAdapter adapter, WGPUSupportedLimits* limits)
```

Comment:
What other kind of bool is there?

Reply:
They typedef’ed this as an int, not sure why